letsencrypt/renewer.py

```diff
@@ -133,8 +150,12 @@ def main(config=None, args=sys.argv[1:]):
 
     zope.component.provideUtility(display_util.FileDisplay(sys.stdout))
 
-    cli_config = configuration.RenewerConfiguration(
-        _create_parser().parse_args(args))
+    args = _create_parser().parse_args(cli_args)
+
+    le_util.make_or_verify_dir(args.logs_dir, 0o700, os.geteuid())
+    cli.setup_logging(args, _cli_log_handler, logfile='renewer.log')
```

Comment:
`StreamHandler` just sends stuff to `STDERR`. I don't see how it could cause any troubles... If someone wants to get rid of it they can just `letsencrypt-renewer 2>/dev/null`.